package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bitdotioinc/node-bitdotio",
-  "version": "0.0.1",
+  "version": "0.1.1",
```

Comment:
assume that only major versions are backward incompatible? aka what's our versioning guarantee/policy?

Reply:
we're using semver, i'll add a note to this readme and the python one